product-service/src/handlers/getProductById/index.js

```diff
@@ -1,17 +1,30 @@
-import {getProductById as getProductByIdService} from "../../services/products";
+import ProductService from "../../services/ProductService";
+import {Product as ProductModel} from '../../models';
 import createResponse from "../../utils/createResponse";
 
+const productService = new ProductService(ProductModel);
+
 export const getProductById = async (event) => {
-    const id = event && event.pathParameters && event.pathParameters.id;
-    const product = getProductByIdService(id);
+    try {
+        const id = event && event.pathParameters && event.pathParameters.id;
```

Comment:
this handler should works only with parameters. Do you have to check for parameters?

Reply:
Hm, didn't understand you. I've grabbed id from pathParametrs and then check it with the condition below